src/coreclr/src/pal/src/loader/module.cpp

```diff
@@ -1676,7 +1697,8 @@ static HMODULE LOADLoadLibrary(LPCSTR shortAsciiName, BOOL fDynamic)
     HMODULE module = nullptr;
     NATIVE_LIBRARY_HANDLE dl_handle = nullptr;
 
-    shortAsciiName = FixLibCName(shortAsciiName);
+    if (shortAsciiName != nullptr)
```

Comment:
That happens when loading embedded coreclr, since it has no path. - the `g_running_in_exe` case.
`FixLibCName` can't handle `nullptr`, nor it needs to. I will update the comment.